app/ReuseNode.php

```diff
@@ -42,4 +42,16 @@ public function cityMerge()
     {
         return $this->hasMany('CityMerge');
     }
+
+    public static function sources() {
+        return self::where("is_source", true)->get();
+    }
+
+    public static function destinations() {
+        return self::where("is_destination", true)->get();
+    }
+
+    public static function fixtures() {
```

Comment:
what is the purpose of having this function since it does the same thing in the function destinations().

Reply:
Good catch! That needs to return `return self::where("is_fixture", true)->get();`, I'll make the change in a bit.